base/multidimensional.jl

```diff
@@ -489,14 +489,14 @@ end
     for T in indexes.parameters
         T <: CartesianIndex ? (M += length(T)) : (M += 1)
     end
-    index_length_expr = index <: Colon ? Symbol(string("Istride_", N+1)) : :(length(index))
+    index_length_expr = index <: Colon ? Symbol("Istride_", N+1) : :(length(index))
```

Comment:
I take it back, the above should work. Do you want me to amend the commit message (to remove the mistyped stevengj's at handle)?